src/sentry/static/sentry/app/components/subscribeButton.tsx

```diff
@@ -13,35 +12,14 @@ type Props = {
 };
 
 export default class SubscribeButton extends React.Component<Props> {
-  static propTypes = {
-    isSubscribed: PropTypes.bool,
-    onClick: PropTypes.func.isRequired,
-    disabled: PropTypes.bool,
-    // `Object is possibly 'undefined'` if we try to use Button.propTypes.size
-    size: PropTypes.any,
-  };
-
   render() {
     const {size, isSubscribed, onClick, disabled} = this.props;
+    const icon = <IconBell color={isSubscribed ? 'blue' : 'black'} />;
```

Comment:
Just a note this component is also used in Issue Details view

![image](https://user-images.githubusercontent.com/79684/83084260-4cca5880-a03d-11ea-81c6-b8778b2fd0aa.png)

Reply:
Talked with @robinrendle about this changing on both pages, seems okay. I've got a follow up ticket to move the subscribe button up with the other controls.